crt_portal/cts_forms/response_templates/crt_auto_response.md

```diff
@@ -2,55 +2,52 @@
 
+Thank you for submitting a report to the Civil Rights Division. Please save your record number for tracking. Your record number is: **{{ record_locator }}**.
 
 If you reported an incident where you or someone else has experienced or is still experiencing physical harm or violence, or are in immediate danger, please call 911 and contact the police.
 
-
----------------------------------
-
-
-WHAT TO EXPECT
-1. We review your report
+# What to Expect
+## 1. We review your report 
 Our specialists in the Civil Rights Division carefully read every report to identify civil rights violations, spot trends, and determine if we have authority to help with your report.
 
 
-2. Our specialists determine the next steps
+## 2. Our specialists determine the next steps
 We may decide to:
+
 - Open an investigation or take some other action within the legal authority of the Justice Department.
 - Collect more information before we can look into your report.
 - Recommend another government agency that can properly look into your report. If so, we’ll let you know.
-In some cases, we may determine that we don’t have legal authority to handle your report and will recommend that you seek help from a private lawyer or local legal aid organization.
 
+In some cases, we may determine that we don’t have legal authority to handle your report and will recommend that you seek help from a private lawyer or local legal aid organization.
 
-3. When possible, we will follow up with you.
-We do our best to let you know about the outcome of our review. However, we may not always be able to provide you with updates because:
-* We’re actively working on an investigation or case related to your report.
-* We’re receiving and actively reviewing many requests at the same time.
-If we are able to respond, we will contact you using the contact information you provided in this report. Depending on the type of report, response times can vary. If you need to reach us about your report, please refer to your report number when contacting us. This is how we keep track of your submission.
 
+## 3. When possible, we will follow up with you
+We do our best to let you know about the outcome of our review. However, we may not always be able to provide you with updates because: 
 
----------------------------------
+- We’re actively working on an investigation or case related to your report. 
+- We’re receiving and actively reviewing many requests at the same time.
 
+If we are able to respond, we will contact you using the contact information you provided in this report. Depending on the type of report, response times can vary. If you need to reach us about your report, please refer to your report number when contacting us. This is how we keep track of your submission.
 
-WHAT YOU CAN DO NEXT
-1. Contact local legal aid organizations or a lawyer if you haven’t already
+# What You Can Do Next
```

Comment:
Are multiple H1's allowed in HTML emails?

Reply:
Good question!  Should we ask Terry?

Reply:
Probably. Seems like there might be a way to have multiple h1's on a page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#multiple_h1_elements_on_one_page

But it's not recommended best practice.

Reply:
@chinelo-ikejimba, what do you think about multiple h1s?

Reply:
@billyhunt the style sheet should allow for styling non-h1 header elements like h1's via the h1 class. This would require reconfiguring the markup of the page a bit, you'd have to bump things down a level, but you should be able to make it look ok provided you're using a USWDS stylesheet.

Reply:
@billyhunt @JackRyan1989 

I understand the concern but in the [design handoff for the html email](https://www.figma.com/file/g9rb8IUGcgiJNJUNLrwG1T/CRT-Reporting-Portal-Designs-and-Handoff?node-id=777%3A7533) section headings are 22px, which makes them H3 per our the [CRT Typography design system](https://www.figma.com/file/yKSv3BjDc9d0rXEe99bjoB/CRT-and-ADA-Styles-Guide_June2021?node-id=0%3A2), whereas the H1 is 45px. Looks like we're good here :)

Reply:
I think the way it is set up we are using the H1 tag but conforming it to the H3 sizing.  It sounds like this is incorrect.  The downside of changing it is it will effect our existing HTML emails (currently just HCE I believe).  @JackRyan1989 sounds like we should make the changes?  We want to make sure we are being fully accessible.